### PR TITLE
#184948254 implementing two factor Authentication 

### PR DESCRIPTION
**Pull request by wilbrord2**
#### What does this PR do?
         This PR is for Authenticating user with a role of seller before login 
#### Description of Task to be completed?
- [x]  Design layout of page in figma 
- [x]  create 2FA page 
- [x] validate form 
- [x] integrating with Api
- [x] testing 

#### How should this be manually tested?

-  In your project terminal, run git fetch origin pull/17/head:ft-two-factor-authentication-#184948254
- Checkout to ft-two-factor-authentication-#184948254 branch
- Install all the dependencies by running npm install
- Then run the project by typing npm run dev
- Navigate to `/authentication`

### routes to be tested
  `/authentication` 

#### What are the relevant pivotal tracker/Trello stories?
Story ID: [#184948254](https://www.pivotaltracker.com/story/show/184948254)

#### Screenshots (if appropriate)
Screenshot (174)
Screenshot (177)
Screenshot (180)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.scss-styled {
+  color: green;
+}/*# sourceMappingURL=style.css.map */
```
Comment (Emile-SHUMBUSHO) [inline src/style.css:3]: remove an empty line of of code

File: src/components/Authentication.jsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+						</span>
+					</div>
+					<AuthForm/>
+					
+				</div>
+			</div>
+		</section>
+    );
+};
+export default sellerAuth;
```
Comment (Emile-SHUMBUSHO) [inline src/components/Authentication.jsx]: Remove an empty line of code

File: src/assets/css/authentication.scss [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+@media (max-width: 767px) {
+    .Container {
+        width: 60%;
+    }
+  }
+  @media (max-width: 467px) {
+    .Container {
+        width: 90%;
+    }
+  }
```
Comment (Emile-SHUMBUSHO) [inline src/assets/css/authentication.scss:141]: Remove an empty line of code

File: src/assets/css/authentication.css [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+@media (max-width: 767px) {
+  .Container {
+    width: 60%;
+  }
+}
+@media (max-width: 467px) {
+  .Container {
+    width: 90%;
+  }
+}/*# sourceMappingURL=authentication.css.map */
```
Comment (Emile-SHUMBUSHO) [inline src/assets/css/authentication.css:141]: Remove an empty line of code

**Comment by Emile-SHUMBUSHO** [comment]
This PR need some test and integration with API in order to test it's fully functionality

**Review by GasanaJr — CHANGES_REQUESTED** [review]
this looks nice. Add some unit tests

**Comment by Emile-SHUMBUSHO** [comment]
Provide a screenshot of your passing tests

**Review by GasanaJr — APPROVED** [review]

**Review by khevin2 — APPROVED** [review]

**Review by GasanaJr — APPROVED** [review]

**Review by Emile-SHUMBUSHO — APPROVED** [review]

**Review by ericndungutse — APPROVED** [review]

**Review by ericndungutse — APPROVED** [review]

**Review by ericndungutse — APPROVED** [review]

**Review by Habinezajanvier — APPROVED** [review]
👍

**Pull request merged**